uc_store/uc_store.install

```diff
@@ -218,6 +224,21 @@ function uc_store_update_last_removed() {
 /**
  * Implements hook_update_N().
  */
+function uc_store_update_1001() {
```

Comment:
Also, please put this update hook at the bottom, in numerical order. Right now it was pasted *before* uc_store_update_1000.

Reply:
Update hook moved to bottom.